scripts/hostapd/uninstall

```diff
@@ -25,3 +25,4 @@ sudo rm -f /etc/hostapd/ca.pem
 sudo rm -f /etc/hostapd/server.pem
 sudo rm -f /etc/hostapd/nat.sh
 sudo rm -f /etc/systemd/system/hostapd.service
+sudo rm -rf /etc/udev/rules.d/70-persistent-net.rules || true
```

Comment:
Does this need `|| true` when `-rf` is used?

Reply:
In case files does not exist for some reason.

Reply:
I mean this:

```
b@deprecated:~$ rm -r stuff
rm: cannot remove 'stuff': No such file or directory
b@deprecated:~$ rm -rf stuff
b@deprecated:~$ echo $?
0
```

It always returns success with the `-f` :)

Reply:
Not blocking, but maybe remove it in the future since it's unnecessary?

Reply:
hmmm interesting